include/picongpu/CMakeLists.txt

```diff
@@ -142,8 +142,9 @@ set(LIBS ${LIBS} ZLIB::ZLIB)
 # Find math from stdlib
 ################################################################################
 
-if(NOT WIN32)
+if((NOT WIN32) AND NOT ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel") )
```

Comment:
QUESTION:
Why did you add perentesis arounf `NOT WIN32` but not around `(NOT ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel"))` ?

Reply:
the cmake documentation was not very clear, whether `NOT` is stronger than `AND`. However in the second case it doesn't matter. ;)

Reply:
Please add `()` also around the second term.